docs/litmus_scenarios.md

```diff
@@ -0,0 +1,20 @@
+### Litmus Scenarios
+Kraken consumes [Litmus](https://github.com/litmuschaos/litmus) under the hood for some infrastructure, pod, and node scenarios 
+
+#### Litmus chaos scenarios
+There are 3 custom resources that are created during each litmus scenario. Below is a description of the resources:
+* ChaosEngine: A resource to link a Kubernetes application or Kubernetes node to a ChaosExperiment. ChaosEngine is watched by Litmus' Chaos-Operator which then invokes Chaos-Experiments
+* ChaosExperiment: A resource to group the configuration parameters of a chaos experiment. ChaosExperiment CRs are created by the operator when experiments are invoked by ChaosEngine.
+* ChaosResult : A resource to hold the results of a chaos-experiment. The Chaos-exporter reads the results and exports the metrics into a configured Prometheus server.
```

Comment:
Thoughts on using the in-cluster prometheus by default to ship the metrics in case of OpenShift?

Reply:
I think this is built into litmus. Would have to be on their side? Would need some clarification on this part